.eslintrc.json

```diff
@@ -48,9 +40,6 @@
         "@typescript-eslint/no-var-requires": "warn",
         "@typescript-eslint/explicit-function-return-type": "off",
         "@typescript-eslint/no-unused-vars": ["warn", { "vars": "all","args": "all", "argsIgnorePattern": "^_" }],
-        "react/no-children-prop": "off",
-        "react-hooks/exhaustive-deps": "off",
```

Comment:
Rart å ha denne med bare for å skru den av. Men jeg mener egentlig at den burde være med som `"plugin:react-hooks/recommended"` og ikke overstyrt

Reply:
Enig! Lager et kort på å se på Eslint regler på tvers av alle apper jeg